code/modules/projectiles/guns/projectile.dm

```diff
@@ -225,7 +226,7 @@
 	..()
 	if(!chambered && stored_magazine && !stored_magazine.ammo_count() && gun_flags &AUTOMAGDROP) //auto_mag_drop decides whether or not the mag is dropped once it empties
 		RemoveMag(user)
-		playsound(user, 'sound/weapons/smg_empty_alarm.ogg', 40, 1)
+		playsound(user, mag_drop_sound, 40, 1)
```

Comment:
Weird, then the inbuilt Byond `sound()` proc accepts nulls and gives an empty sound in return, for some reason.